packages/aserto/src/aserto/client/directory/__init__.py

```diff
@@ -1,14 +1,31 @@
-from typing import List, Literal, Tuple
+from typing import List, Literal, Tuple, Any
 
-from grpc import ChannelCredentials, ssl_channel_credentials
+from grpc import ChannelCredentials, ssl_channel_credentials, secure_channel
 
 
 class NotFoundError(Exception):
     pass
 
+class InvalidArgument(Exception):
+    pass
+
+class NilClient(Exception):
+    pass
 
 Header = Literal["authorization", "aserto-tenant-id", "if-match", "if-none-match"]
 
+def validate_addresses(
+    address: str,
+    reader_address: str,
+    writer_address: str,
+    importer_address: str,
+    exporter_address: str,
+    model_address: str) -> Any:
+    if address == "" and reader_address == "" and writer_address == "" and importer_address == "" and exporter_address == "" and model_address == "":
+        return InvalidArgument
```

Comment:
Python has a built in `ValueError` exception that is used to indicate invalid inputs. We should use that instead.

Reply:
It would also be good to pass a message that explains the issue and to raise the exception instead of returning it:
```py
if ...:
    raise ValueError("at least one directory service address must be specified")
```